### PR TITLE
Dev-5646 Amount Chart Updates

### PR DESCRIPTION
**Pull request by jonhill13**
**High level description:**

Updates chart to new mock.

**Technical details:**

Technical details for the knowledge of other developers.

**JIRA Ticket:**
[DEV-5646](https://federal-spending-transparency.atlassian.net/browse/DEV-5646)

**Mockup:**
https://bahdigital.invisionapp.com/share/BCIAEH6UVTR#/screens/296044453

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
- [N/A] Verified mobile/tablet/desktop/monitor responsiveness
- [N/A] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [N/A] Added Unit Tests for methods in Container Components, reducers, helper functions, and models `if applicable`
- [N/A] All `componentWillReceiveProps` and `componentWillMount` in relevant child/parent components/containers replaced with [future compatible life-cycle methods](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html) per [JIRA item 3277](https://federal-spending-transparency.atlassian.net/browse/DEV-3277)
- [N/A] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [N/A] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [N/A] Design review complete `if applicable`
- [N/A] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [x] Code review complete

### CODE REVIEW

File: src/js/components/covid19/AmountsVisualization.jsx [hunk 1]
```diff
@@ -39,8 +40,10 @@ const AmountsVisualization = ({
     const [remainingBalanceRectangleData, setRemainingBalanceRectangleData] = useState(null);
     const [totalLineData, setTotalLineData] = useState(null);
     const [outlayLineData, setOutlayLineData] = useState(null);
-    const [obligationLineData, setObligationLineData] = useState(null);
-    const [remainingBalanceLineData, setRemainingBalanceLineData] = useState(null);
+    const [obligationLineDataOne, setObligationLineDataOne] = useState(null);
```
Comment (ebdabbs) [inline src/js/components/covid19/AmountsVisualization.jsx:43]: Do these represent 4 different scenarios? Either add a comment or use a more descriptive name to improve readability. Inspecting the viz from the browser, it looks like these lines stack up to form what visually looks like a single line, so I'm not following why it's broken out into four pieces.

Comment (jonhill13) [reply src/js/components/covid19/AmountsVisualization.jsx:43]: The one remaining balance line has been broke up into four different lines due to the overlap captured in the mock.

https://bahdigital.invisionapp.com/share/BCIAEH6UVTR#/screens/296044453

Comment (ebdabbs) [reply src/js/components/covid19/AmountsVisualization.jsx:43]: Makes sense - thank you! Can you leave a comment to that effect? Something like `the part of the line that can overlap with Total Remaining Balance text`, etc.

**Comment by jonhill13** [comment]
@ebdabbs I noticed issues as well when using fake data.  I do not know what is causing those issues. 

With real data it seems to be working.
Screen Shot 2020-07-29 at 11 54 21 AM
Screen Shot 2020-07-29 at 11 54 34 AM

**Comment by ebdabbs** [comment]
Weird. My concern is that the real data will change over time, and we could run into this scenario in production. I can merge these changes, since they are an improvement, and make a follow-up ticket.

**Pull request merged**